chatcommands.lua

```diff
@@ -286,6 +286,30 @@ minetest.register_chatcommand("area_open", {
 })
 
 
+if factions then
```

Comment:
Please use https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L5105

If playerfactions is not enabled, Minetest/Lua will warn about accessing an undefined variable.
`if minetest.global_exists("factions") then`